### ISSUE TITLE
Status.php - Need to make a history graph

### ISSUE THREAD

**Issue by mitzey234** [mitzey234/Gameslab-Server-Manager#2]
We need to make a history graph of the memory and CPU usage. There are some examples of the bar graphs on the dashboard page on the index.php page.

**Comment by Candunc** [comment]
Eh... this is out of the scope for a client-side addition. We'd need to have a full-time script running on the host computer saving the relevant data to a database.

**Comment by mitzey234** [comment]
That is true. But what I did have in mind was something like the kind the McMyAdmin has. The web interface (once you log in) starts placing recently taken memory data. And every 2 sec from login it will place the current data for the memory onto a graph. Same thing with the CPU.

**Comment by Candunc** [comment]
Ok, that would be more do-able
